editor.planx.uk/src/@planx/components/PlanningConstraints/List.tsx

```diff
@@ -198,14 +200,31 @@ function ConstraintListItem({ children, ...props }: ConstraintListItemProps) {
           expandIcon={<Caret />}
           sx={{ pr: 1.5, background: `rgba(255, 255, 255, 0.8)` }}
         >
-          <Typography
-            component="div"
-            variant="body2"
-            pr={1.5}
-            sx={{ color: allEntitiesInaccurate ? "GrayText" : "inherit" }}
-          >
-            {children}
-          </Typography>
+          <Box sx={{ display: "flex", justifyContent: "space-between", alignItems: "center", width: "100%", }}> 
+            <Typography
+              component="div"
+              variant="body2"
+              pr={1.5}
+            >
+              {children}
+            </Typography>
+            {allEntitiesInaccurate && (
+              <>
+                <Chip
+                  label="Not applicable"
+                  variant="notApplicableTag"
+                  size="small"
+                  sx={{ mr: 0.75, ":not(#negative-constraints-list &)": { display: "none !important" } }}
```

Comment:
Here's how I'd have this with props and not CSS - 

- Add an additional prop to `ConstraintsListProps` (something like `variant: "positive" | "negative"` maybe?)
- Populate this x2 in `Public.tsx`
- Drill the same prop down into `ConstraintListItem`
- Use this prop to conditionally render this chip

Reply:
FYI the existing boolean prop `value` on `ConstraintsListItemProps` already indicates applicable / not applicable - I don't think you should have to add a new one here !

Reply:
Oh nice, I missed this!

Reply:
Thanks both, I've got this working based on the `value` prop 👍